include/SDL3/SDL_events.h

```diff
@@ -714,67 +715,119 @@ typedef struct SDL_TouchFingerEvent
     SDL_WindowID windowID; /**< The window underneath the finger, if any */
 } SDL_TouchFingerEvent;
 
-
 /**
- * Pressure-sensitive pen touched or stopped touching surface (event.ptip.*)
+ * Pressure-sensitive pen proximity event structure (event.pmotion.*)
+ *
+ * When a pen becomes visible to the system (it is close enough to a tablet,
+ * etc), SDL will send an SDL_EVENT_PEN_PROXIMITY_IN event with the new
+ * pen's ID. This ID is valid until the pen leaves proximity again (has
+ * been removed from the tablet's area, the tablet has been unplugged, etc).
+ * If the same pen reenters proximity again, it will be given a new ID.
+ *
+ * Note that "proximity" means "close enough for the tablet to know the tool
+ * is there" and that the pen touching and lifting off from the tablet while
+ * not leaving the area is a different event.
```

Comment:
What event is that?

Reply:
SDL_EVENT_PEN_DOWN and SDL_EVENT_PEN_UP, I'll update the text.

Reply:
Fixed in 0ff84e6b8a42d5282ea442ea264409f1698d8f2f.